src/frontend/classic/components/MenuLeftPanel.svelte

```diff
@@ -0,0 +1,283 @@
+<script lang="ts">
+    import {
+        SideNav,
+        SideNavItems,
+        SideNavMenu,
+        SideNavLink,
+    } from "carbon-components-svelte";
+    import LogoDiscord16 from "carbon-icons-svelte/lib/LogoDiscord16";
+    import Home16 from "carbon-icons-svelte/lib/Home16";
+    import LogoGithub16 from "carbon-icons-svelte/lib/LogoGithub16";
+    import App16 from "carbon-icons-svelte/lib/App16";
+    import Doc16 from "carbon-icons-svelte/lib/Doc16";
+    import Events16 from "carbon-icons-svelte/lib/Events16"; // Maintainers
+    import EventsAlt16 from "carbon-icons-svelte/lib/EventsAlt16"; // contributors
+    import Debug16 from "carbon-icons-svelte/lib/Debug16";
+    import Image16 from "carbon-icons-svelte/lib/Image16";
+    import Location16 from "carbon-icons-svelte/lib/Location16";
+    import { themes } from "./Theme.svelte";
+    import MenuLeftPanelItem from "./MenuLeftPanelItem.svelte";
+    import MenuLeftPanelSelect from "./MenuLeftPanelSelect.svelte";
+    import MenuLeftPanelInput from "./MenuLeftPanelInput.svelte";
+    import MenuLeftPanelToggle from "./MenuLeftPanelToggle.svelte";
+    import { getSettingDefaultValue, storageKeys } from "../utils/storage";
+    export let isSideNavOpen = false;
+    export let changeUIMode: () => void;
+    export let changeTheme: (themeId: string) => void;
+
+    function openExternalLink(uri: string) {
+        nw.Shell.openExternal(uri);
+    }
+
+    const win = nw.Window.get();
+
+    $: sideNavStyle = isSideNavOpen ? `min-width: ${win.width / 2}px;` : "";
```

Comment:
Feels too wide. Wasn't the default value enougth ? I feel like the global UI flow is now disturbed.

Reply:
It's too large when the window is extended, but when you have a small window it wasn't wide enough

Reply:
Ok my bad I remove the style and it's working nice